src/backend/binary.rs

```diff
@@ -42,7 +42,7 @@ impl<'a> Binary<'a> {
             return Err(anyhow!("unsupported ELF header: e_machine"));
         }
 
-        if elf.header.e_version != EV_CURRENT.into() {
+        if elf.header.e_version != EV_CURRENT as u32 {
```

Comment:
This change is necessary because serde_json defines a conflicting `Into` impl in this case.